test/test_parse.py

```diff
@@ -88,6 +88,13 @@ def test_extract_number(self):
         self.assertEqual(extract_number("three quarter cups"), 3.0 / 4.0)
         self.assertEqual(extract_number("three quarters cups"), 3.0 / 4.0)
         self.assertEqual(extract_number("twenty two"), 22)
+        self.assertEqual(extract_number(
+            "Twenty two with a leading capital letter"), 22)
+        self.assertEqual(extract_number(
+            "twenty Two with two capital letters"), 22)
```

Comment:
i think you meant to write Twenty Two in here

Reply:
Good catch, though I was actually going for, "twenty Two with Two..." I'll fix that.